include/picongpu/param/transitionRadiationObserver.param

```diff
@@ -0,0 +1,123 @@
+ */
+
+#pragma once
+
+namespace picongpu
+{
+namespace transitionRadiation
+{
+    /** Compute observation angles
+     *
+     * This function is used in the Radiation plug-in kernel to compute
+     * the observation directions given as a unit vector pointing
+     * towards a 'virtual' detector
+     *
+     * This default setup is an example of a 2D detector array. It computes
+     * observation directions for 2D virtual detector field
+     * with its center pointing toward the +y direction (for theta=0, phi=0)
+     * with observation angles ranging from
+     * theta = [angle_theta_start : angle_theta_end]
+     * phi   = [angle_phi_start   : angle_phi_end  ]
+     * Every observation_id_extern index moves the phi angle from its
+     * start value toward its end value until the observation_id_extern
+     * reaches N_split. After that the theta angle moves further from its
+     * start value towards its end value while phi is reset to its start
+     * value.
+     *
+     * The unit vector pointing towards the observing virtual detector
+     * can be described using theta and phi by:
+     * x_value = sin(theta) * cos(phi)
+     * y_value = cos(theta)
+     * z_value = sin(theta) * sin(phi)
+     * These are the standard spherical coordinates.
+     *
+     * The example setup describes an detector array of
+     * 256x256 detectors ranging from 0 to pi for the azimuth angle
+     * theta and from 0 to 2 pi for the polar angle phi.
+     *
+     * @param    observation_id_extern
+     *           int index that identifies each block on the GPU
+     *           to compute the observation direction
+     *
+     * @return   unit vector pointing in observation direction
+     *           type: vector_64
+     *
+     */
+
+    HDINLINE float3_X observationDirection(const int observation_id_extern)
+    {
+        /* generate two indices from single block index */
+        /** split distance of given index
+         * pseudo-code:
+         * index_a = index / split_distance
+         * index_b = index % split_distance
+         */
+        constexpr int N_angle_split = parameters::nPhi;
+        /** get index for computing angle theta: */
+        const int my_index_theta = observation_id_extern / N_angle_split;
+        /** get index for computing angle phi: */
+        const int my_index_phi = observation_id_extern % N_angle_split;
+
+
+        /* set up observation angle range */
+        /* angles range for theta */
+        /** minimum theta angle [rad] */
+        const picongpu::float_64 angle_theta_start = parameters::thetaMin;
+        /** maximum theta angle [rad] */
+        const picongpu::float_64 angle_theta_end = parameters::thetaMax;
+        /* angles range for phi */
+        /** minimum phi angle [rad] */
+        constexpr picongpu::float_64 angle_phi_start = parameters::phiMin;
+        /** maximum phi angle [rad] */
+        constexpr picongpu::float_64 angle_phi_end = parameters::phiMax;
```

Comment:
If you moved the definition of the angle ranges to the other param file, you do not need a renaming here.